frontend/src/app/models/income-source.command.ts

```diff
@@ -0,0 +1,5 @@
+export interface IncomeSourceCommand {
```

Comment:
I like it that the frontend interfaces match with the backend interfaces. And I'm in the middle of a refactoring on persons and income source types to use commands, too.

Commands look like an unnecessary burden when the command looks exactly like the model except for the absent ID. But my experience has shown that when the model becomes more complex, they start differing:

 - models have associations modeled as embedded objects containing IDs and names (like an income source has an income source type object), whereas commands only have IDs of related entities
 - models contain a large piece of information, whereas comands are usually smaller because you don't provide all the details when creating/updating the entity
 - etc.

I agree that it looks like repetition, but I don't think forgetting a refactoring of one when refactoring the other can be unnoticed: the backend will slap you in the face.